src/components/ProfileView/ProfileTemplateList.tsx

```diff
@@ -12,7 +12,7 @@ import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 import ExpandLessIcon from '@mui/icons-material/ExpandLess';
 import IconButton from '@mui/material/IconButton';
 import { useState } from 'react';
-import { DeleteTemplateButton } from '@/components/DeleteTemplate/DeleteTemplateButton';
+import { DeleteTemplateButton } from '@/components/deleteTemplate/DeleteTemplateButton';
```

Comment:
I had to capitalize the DeleteTemplate because all the directories in components are Capitalized. Did you change the name of the directory aswell? If so, why?

Reply:
it must have gotten uncapitalised with the merge somehow, i'll fix it asap 👍